src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/LocalStackIntegrationTest.java

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Yes, I created another test because first I had some issues with Minio, so I was testing with another.
Finally figured out the problem. I left this new test, but it can be removed if it's really painful to have more tests/

Reply:
What issues? The MinIO tests should be run in CI (unlike the ones using AWS).

Reply:
Because there is a need now to have configuration such this https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/619/files#diff-284b8c644860bd2e66326c2c397622c66e094754eee8e4f003fdae0c863799f1R233 
Interesting as this was not working with test using TestContainer while it was working using an external running MinIO
so I didn't figured out this problem while I didn't have same issue with LocalStack.

Reply:
Not sure I follow. So long as you have Docker installed, you should not need to do anything beyond

```bash
mvn test -Dtest=MinioIntegrationTest
```

Reply:
Comment this line here: https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/619/files#diff-284b8c644860bd2e66326c2c397622c66e094754eee8e4f003fdae0c863799f1R237
```
//presignerBuilder.serviceConfiguration(s3Configuration);
```

Now run:
``` 
mvn clean test -Dtest=MinioIntegrationTest
```
Then:
```
mvn test -Dtest=LocalStackIntegrationTest
```
It just took me time to figure out this configuration. At the start, I was thinking maybe Minio doesn't support AWS SDK v2, so I tested with another impl.